src/commands/cork_proposal/cellar_v2_2/force_position_out.rs

```diff
@@ -0,0 +1,76 @@
+use abscissa_core::{clap::Parser, Command, Runnable};
+
+use crate::{
+    application::APP,
+    cellars,
+    commands::cork_proposal::print_proposal,
+    prelude::*,
+    proto::{
+        cellar_v2_2governance::{ForcePositionOut, Function},
+        governance_call::Call,
+        CellarV22governance, GovernanceCall,
+    },
+};
+
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nCalls forcePositionOut() on the target V2.2 cellar contract at the specified block height.\nFor more information see https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol"
+)]
+pub struct ForcePositionOutCmd {
+    #[clap(short = 'p', long)]
+    /// The ID of the position
+    position_id: u32,
+
+    #[clap(short, long)]
+    /// The index of the position
+    index: u32,
+
+    /// Whether the position is a debt position
+    debt: bool,
```

Comment:
Does this need a `clap` macro? Also, the field refers to `in_debt_array`, is this shortened intentionally?